services/platform/platform_test.go

```diff
@@ -1,105 +0,0 @@
-// pmm-managed
```

Comment:
removed these tests since they are in place of unit tests and they are communication with an external service, which make them a bit confusing.

We already tests for this in our API tests